.travis.yml

```diff
@@ -14,6 +14,8 @@ branches:
     - master
 
 env:
+  - DJANGO="django>=2.2,<3.0"
+  - DJANGO="django>=3.0,<3.3"
```

Comment:
add a blank line after that line

Reply:
I think it's not longer relevant, please see how it is now